wifiphisher/common/phishinghttp.py

```diff
@@ -20,6 +20,24 @@ def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
+class ValidateHandler(tornado.web.RequestHandler):
+
+    def initialize(self, em):
+        self.em = em
+
+    def get(self, cred_type, cred_content):
+        """
+        Override the get method
+
+        :param self: A tornado.web.RequestHandler object
+        :type self: tornado.web.RequestHandler
+        :return: None
+        :rtype: None
+        """
+        value = self.em.verify_cred((cred_type, cred_content))
+        self.write("%s" % value)
+
```

Comment:
@sophron 
I'm thinking for a while about this PR tonight. Do you think we should pass a validating function to the template directly, so we can just make decision how to display the web page base on the result of the verification?

Like the following link: [ui_module](https://stackoverflow.com/questions/12993835/passing-a-custom-python-function-into-a-tornado-template)